trieste/acquisition/function.py

```diff
@@ -1060,7 +1060,7 @@ def prepare_acquisition_function(
         feasible_mean, _ = objective_model.predict(feasible_query_points)
 
         _pf = Pareto(feasible_mean)
-        _reference_pt = get_reference_point(_pf.front)
+        _reference_pt = get_reference_point(feasible_mean)
```

Comment:
Just for learning curious: as `get_reference_point` is currently only intend to extract the reference point from a 'front' (as conveyed from its input arg's name, though we never check if input is really a 'front'), is 1063 more like a tmp fix, or some ref doc could be mentioned here as well to align with this adaptive choice of ref pt in CMOO? And if there is only 1 feasible point, we'd assume this behaves the same?

Reply:
This is a temporary fix. It's possibly a very bad choice in general. On my current problem it works well because the constraints are quite restrictive. We may want to discuss what we want to do with reference point management in general. It feels that we can't stick to a single solution.